flask_autoindex/entry.py

```diff
@@ -251,7 +252,10 @@ def asc():
         entries = []
         for name in dirlist:
             try:
-                entries.append(Entry(os.path.join(self.path, name), rootdir))
+                if NT_OS:
+                    entries.append(Entry(os.path.join(self.path, name).replace('\\', '/'), rootdir))
```

Comment:
How about using `os.path.sep` instead of `if NT_OS`? `os.path.sep` looks to be `'/'` in Linux but `'\\'` in Windows.